### DISCUSSION TITLE
Attribute [webhook] does not exist.

### DISCUSSION THREAD

**Discussion by hamza-saqib** [spatie/laravel-webhook-client#174] [Q&A]
i have php 7.3 and laravel 7 installed. i have installed both client and server package of this. but while it installed successfully but after few days later when i did `composer install` than i got this even i'm getting this on clearing cache. i have instlled this client package with command `composer require spatie/laravel-webhook-client:"*"` because it was causing version dependency issue.

my route:  `Route::webhooks('echeck-webhook-receiving');`

   ```
InvalidArgumentException 

  Attribute [webhook] does not exist.

  at vendor/laravel/framework/src/Illuminate/Routing/RouteRegistrar.php:93
    89|      */
    90|     public function attribute($key, $value)
    91|     {
    92|         if (! in_array($key, $this->allowedAttributes)) {
  > 93|             throw new InvalidArgumentException("Attribute [{$key}] does not exist.");
    94|         }
    95| 
    96|         $this->attributes[Arr::get($this->aliases, $key, $key)] = $value;
    97| 

      +2 vendor frames 
  3   routes/web.php:172
      Illuminate\Support\Facades\Facade::__callStatic("webhook")
```

**Comment by jasonziolo** [comment]
I'm stuck on this as well, any solutions?

**Reply by michaelklopf** [reply]
Depends on which version you are using. Version 3 of the package is not compatible with PHP 7.3 anymore, so maybe it's that. 

edit
And version 2 needs 7.4.

**Comment by ken-tam** [comment]
make sure you import the proper Route, use Illuminate\Support\Facades\Route;

I had an empty web.php and imported the use Illuminate\Routing\Route would cause the issue.
